packages/launchpad/src/index.ts

```diff
@@ -1,7 +1,18 @@
+// Re-exports for backwards compatibility
+export {
+  decodeAminoPubkey,
+  decodeBech32Pubkey,
+  encodeAminoPubkey,
+  encodeBech32Pubkey,
+  encodeSecp256k1Pubkey,
+  pubkeyToAddress,
+  pubkeyType,
+  SinglePubkey as PubKey,
+} from "@cosmjs/amino";
+
 import * as logs from "./logs";
 export { logs };
 
-export { pubkeyToAddress, rawSecp256k1PubkeyToAddress } from "./address";
```

Comment:
I removed it because it was a shortcut that does not fit in the more general flow. We only have rawSecp256k1PubkeyTo**Raw**Address now. Have a look at the CHANGELOG entry.